lending_pool/src/router.rs

```diff
@@ -142,6 +143,14 @@ pub trait RouterModule: proxy_common::ProxyCommonModule + factory::FactoryModule
         Ok(())
     }
 
+    #[only_owner]
+    #[endpoint(setAssetLTV)]
+    fn set_asset_ltv(&self, asset: TokenIdentifier, ltv: Self::BigUint) -> SCResult<()> {
```

Comment:
No need to return `SCResult<()>` here at all. I've done this mistake myself in the past.

Reply:
done